content/en/about/contributing/syntax/_index.md

```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
+
+To create new chapters in the documentation (visible in left-side navigation
+tree), you have to create new folders under `/content/<language>` and add a file
+`_index.md` to that folder. The beginning of the markdown file contains a
+[front matter](https://gohugo.io/content-management/front-matter/) in YAML, JSON
+or TOML, holding the metadata of the page:
```

Comment:
Let's only suggest YALM there, we don't want to mix front header types

Reply:
OK, will update. For now I restored the current content.